cmd/api-errors.go

```diff
@@ -2158,7 +2159,8 @@ var errorCodes = errorCodeMap{
 		HTTPStatusCode: http.StatusBadRequest,
 	},
 	ErrCORSRequestNotAllowed: {
-		Code:           "AccessForbidden",
+		Code: "AccessForbidden",
+		// FIXME: 'evalution' is a spelling mistake that is copied verbatim from the S3 API in order to be as compatible as possible.
```

Comment:
We do not need to follow the descriptions letter for letter.

From https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingRESTError.html

> The error message contains a generic description of the error condition in English. It is intended for a human audience. Simple programs display the message directly to the end user if they encounter an error condition they don't know how or don't care to handle.

Please correct spelling mistakes and if there are ways to make the description clearer feel free to do so.

Reply:
Actually this is not going to work @klauspost applications use strings directly in many SDKs and applications by comparing verbatim strings.

I am afraid we can't change it. Unfortunately.